modules/sdk-coin-atom/src/lib/transaction.ts

```diff
@@ -116,26 +115,31 @@ export class Transaction extends BaseTransaction {
    *
    * @param rawTransaction
```

Comment:
Lets be consistent on jdocs
```suggestion
   * @param {string} rawTransaction
```

Reply:
NIt: Also if you can make the other functions in the file consistent with JSDocs that would be awesome